lib/config/presets/internal/helpers.ts

```diff
@@ -22,7 +22,7 @@ export const presets: Record<string, Preset> = {
     extends: [':followTag(typescript, rc)'],
   },
   pinGitHubActionDigests: {
-    description: 'Pin `github-action` digests.',
+    description: 'Pin `github-action` digests.\n In case you use `vX` pins and you\'d like not only pin digests but specify which exact tag is used by changing `vX` to `vX.Y.Z` - except enabling this preset, add to your config [this workaround](https://github.com/renovatebot/.github/blob/f5a5fc056c78fe7741a4fde59ee94feed622cd3e/default.json#L45-L50).',
```

Comment:
```suggestion
    description: 'Pin `github-action` digests.\n In case you use `vX` pins and you\'d like not only pin digests but specify which exact tag is used by changing `vX` to `vX.Y.Z` - except enabling this preset, add to your config [this workaround](https://github.com/renovatebot/renovate/discussions/21901#discussioncomment-6563510).',
```